modules/library/main/src/main/java/org/geotools/filter/function/AndFunction.java

```diff
@@ -0,0 +1,68 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2002-2008, Open Source Geospatial Foundation (OSGeo)
```

Comment:
Dates here should be all just "2020", e.g.:

```suggestion
 *    (C) 2020, Open Source Geospatial Foundation (OSGeo)
```